async_processor/types.py

```diff
@@ -216,6 +216,20 @@ def to_input(self) -> Input:
         return SQSInput(self)
 
 
+class AMQPInputConfig(InputConfig):
+    type: constr(regex=r"^amqp$") = "amqp"
+    queue_url: constr(
```

Comment:
Should this be called `queue_url`? If this is just the URL for rabbitmq, maybe we call it uri/url?

Reply:
Let us not call it queue URL as this URL is not queue specific.